### DISCUSSION TITLE
MonoGame on pre rendered background

### DISCUSSION THREAD

**Discussion by vlcp197** [MonoGame/MonoGame#8602] [Graphics]
I would like to make a game like digimon 1 from psx, with pre rendered backgrounds the 3d models on top of it, but i cant find any tutorials about in monogame. Can someone recommend some material to study?

**Comment by KelsamGames** [comment]
I haven't played that, but if you mean you want a 2D image from your own models so you don't have to render them every frame, all you need to do is draw them to a Render Target (as opposed to the back buffer). You can then use the RT just like any Texture and draw it each frame instead of the models.

If you don't want to include those models in your game at all, you could even make a completely separate project that just renders a frame of your scene and outputs an image for your actual game to consume.

**Reply by vlcp197** [reply]
Thx

**Comment by MonophonicOne** [comment]
I quickly glanced at some gameplay in youtube. I think it mixes large picture backgrounds, or possibly even tilemaps, classical 2D sprites, and 3D models. Any 2D three-quarters perspective tutorial, combined with any 3D model rendering tutorial, should get you most of the way there.

Basically, where you would draw a certain 2D sprite, you instead draw a certain 3D model. Using the same orthographic projection matrix for both is perfectly fine. Unfortunately SpriteBatch doesn't seem to expose the projection matrix it uses, so you may need to check its source code to get things to align well.

**Reply by vlcp197** [reply]
Thanks
